src/file-events/cpp/logging.cpp

```diff
@@ -9,6 +9,10 @@ Logging::Logging(JavaVM* jvm)
     , getLevelMethod(getThreadEnv()->GetStaticMethodID(clsLogger.get(), "getLogLevel", "()I")) {
 }
 
+void Logging::invalidateLogLevelCache() {
+    lastLevelCheck = chrono::steady_clock::time_point();
```

Comment:
What does this do? Create a new timepoint for 0ms? I didn't find documentation on the constructor of `time_point`...

Reply:
> Default constructor, creates a time_point representing the Clock's epoch (i.e., `time_since_epoch()` is zero).

https://en.cppreference.com/w/cpp/chrono/time_point/time_point